contracts/src/v0.8/ccip/test/helpers/CCIPReaderTester.sol

```diff
@@ -1,79 +0,0 @@
-// SPDX-License-Identifier: BUSL-1.1
-pragma solidity 0.8.24;
-
-import {Internal} from "../../libraries/Internal.sol";
-import {OffRamp} from "../../offRamp/OffRamp.sol";
-
-contract CCIPReaderTester {
```

Comment:
Is this safe to remove? iirc the off-chain code was relying on these helper contracts

Reply:
Confirmed offline - this is no longer needed